authui/src/phoneInput.ts

```diff
@@ -55,7 +55,7 @@ function compileCountryList(): PhoneInputCountry[] {
     const countryFlag = getEmojiFlag(countryCode);
     const countryCallingCode = getCountryCallingCode(countryCode);
     return {
-      flagEmoji: `<span class="country-flag-icon phone-input__country-flag">${countryFlag}</span>`,
+      flagEmoji: `<span class="country-flag-icon phone-input__country-flag twemoji-countries">${countryFlag}</span>`,
```

Comment:
What actually is the cause?

Reply:
This one is not related, just think better to apply icon default settings. The cause lies on the font files.